src/resources/Subscriptions.ts

```diff
@@ -74,19 +74,18 @@ export type SubscriptionPlanItem =
     | { planType: InstallmentPlan.FIXED_CYCLES; fixedCycles: number }
     | { planType: InstallmentPlan.FIXED_CYCLE_AMOUNT; fixedCycleAmount: number };
 
-export interface SubscriptionsListParams extends CRUDPaginationParams {
+export type SubscriptionsListParams = CRUDPaginationParams & {
```

Comment:
Updating interfaces to types to keep things a bit more consistent across applications. For most of them there is no reason to use interfaces over types.